src/sentry/logging/handlers.py

```diff
@@ -0,0 +1,61 @@
+"""
+sentry.logging.handlers
+~~~~~~~~~~~~~~~~~~~~~~~
+:copyright: (c) 2010-2016 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+
+from __future__ import absolute_import
+
+import logging
+
+from django.utils.encoding import force_bytes
+
+
+logger = logging.getLogger('sentry.audit')
+
+
+class TuringHandler(logging.StreamHandler):
+    """
+    Decides between logging human or machine readable lines.
+    """
+    def emit(self, record):
+        context = record.args
+        # Check to make sure someone is following the rules.
+        if not context:
+            context = record.msg
+        if isinstance(context, dict):
+            from sentry.options import get
```

Comment:
This is gross but this handler gets instantiated way too soon and it causes a bunch of tests to fail.

Please advise.